flink-runtime/src/main/java/org/apache/flink/runtime/security/SecurityUtils.java

```diff
@@ -230,10 +231,21 @@ private void validate() {
 		}
 
 		private static List<String> parseList(String value) {
-			if(value == null) {
+			if(value == null || value.isEmpty()) {
 				return Collections.emptyList();
 			}
-			return Arrays.asList(value.split(","));
```

Comment:
The wisdom of that paragraph has been debated extensively on stackoverflow.  I'll just say that the `StringTokenizer` is not deprecated and, in my opinion, fits this scenario uniquely well.   Go ahead either way.